app/services/storage/compare/page.tsx

```diff
@@ -15,7 +16,23 @@ export default async function CompareStorageOptions() {
 
   return (
     <>
-      <Hero title={metadata.title} description={metadata.description} />
+      <Hero title={metadata.title} description={metadata.description}>
+        {metadata.links?.map((link: any, index: number) => (
+          <ButtonLink
+            key={index}
+            variant={
+              link.category === "Documentation"
+                ? "primary_filled"
+                : "secondary_filled"
+            }
```

Comment:
rather than having this ternary here, would it make sense in the links part of the mdx file to have an option like "isPrimary: true" or something like that? To check that versus this matching on Documentation?